### PR TITLE
change to get all questions

### PR DESCRIPTION
**Pull request by Hamidreza-Fakharzadeh**

### CODE REVIEW

**Comment by Hamidreza-Fakharzadeh** [comment]
All member of our team worked together on both question and answer feathers.

**Comment by Hamidreza-Fakharzadeh** [comment]
promises has been done for routers

File: backend/routes/questions.js [hunk 1]
```diff
@@ -1,88 +1,43 @@
...
-        } else {
-          return res
-            .status(201)
-            .json({ question: question, newAnswer: newAnswer });
-        }
-      });
-    });
-  });
+// Answer part:
+router.route('/:id/answer').post((req, res) => {
```
Comment (delamorinierejh) [inline backend/routes/questions.js:27]: Hi @Hamidreza-Fakharzadeh - I think if you want to use this chained `.then` approach, then you will have to make sure you are returning a promise inside each `.then` block. Are you sure this working? Is it definitely saving the answer in your database and returning it as an answer to the question when you go to show route for the question? Thanks

**Comment by Hamidreza-Fakharzadeh** [comment]
other pull request for updating

**Pull request closed without merging**